app/views/admin/stats/top_balances/_channels.html.slim

```diff
@@ -0,0 +1,18 @@
+br
+div.panel-body
+  div.adv-table
+    table.display.table.table-bordered.table-striped.dynamic-table id="dynamic-table"
+      tr
+        th Account ID
+        th Balance
+      tbody
+        - @result.each do |entry|
+          tr.gradeX
+            - if entry["account_id"].starts_with?(@table_identifier)
+              - youtube_channel_id = entry["account_id"].split(":")[1]
+              td = link_to @youtube_channels[youtube_channel_id]&.dig(1), admin_publishers_path(@youtube_channels[youtube_channel_id]&.dig(0))
```

Comment:
This should be `admin_publisher_path` same with below. I had some problems locally with not having the latest database so I just wrote some code like this
```
              - youtube_channel = @youtube_channels[entry["account_id"].split(":")[1]]
              td
                - if youtube_channel.present?
                  = link_to(youtube_channel&.dig(1), admin_publisher_path(youtube_channel&.dig(0)))
                - else
                  = entry["account_id"]
                end
```